### ISSUE TITLE
this tool will merge db0~dbn data from source node to target node

### ISSUE THREAD

**Issue by silence0607** [vipshop/redis-migrate-tool#17]
source node: A   target node: B

use redis-migrate-tool to sync data between node A and node B
the output infomation about  `redis-cli keyspace` is:

node A
# Keyspace
db0:keys=1133032,expires=0,avg_ttl=0
db15:keys=1,expires=0,avg_ttl=0

node B
# Keyspace
db0:keys=1133033,expires=0,avg_ttl=0

maybe the tool forget to use `SELECT` command when sync data between two nodes?

**Comment by deep011** [comment]
Yes, rmt will ignore db in the migration because of it is always used to migrating data between redis groups(multiple redis).
If you want just migrate the special db of the source redis group to the special db of the target redis group, i can add 'redis_db' in the rmt config file.

**Comment by silence0607** [comment]
rmt is a excellent tool for redis，I think rmt can be used in many situation not only from multiple nodes to multiple nodes, such like single nodes(sentinel) to single nodes(sentinel). 
So, the different db's data should be migrate to the right db in target node，if you add 'redis_db' in the rmt config file，maybe this is good idea, but I think migrate all db data from source nodes to target nodes(all data not in db0) can be a default option,  thanks!!

best wishes~

**Comment by xuhuihihi** [comment]
When are you add 'redis_db' in the rmt config file?

**Comment by silence0607** [comment]
@deep011 I try to parse the dbid in the rmt_redis.c，but every SELECT command maybe parse；if migrate data between cluster node，it may cause problem，so, add  'redis_db' in config file would be a good idea

**Comment by 929121806** [comment]
需要这个功能。

**Comment by lidezheng** [comment]
已经解决了。
在这里：#40

**Comment by cui-ying** [comment]
@deep011 请问这个指定源redis_db 和 目标redis_db  的功能已经可以使用了么？为什么我用最新的master，还是只能是源所有的db，迁移到了目标的db0.???即使加了这个参数redis_db 也不行。不知道是不是我配置的有错误。哪里有例子能参考下？谢谢！！！

**Comment by aishangnidemei** [comment]
@trying-cui  请问你这个问题解决了没有啊？怎么配置的啊？

**Comment by akovacs123** [comment]
I reckon this is still not available (I experience dumping all stuff into db0). Are you planning to add this feature? As a workaround, do you recommend something?
